.github/workflows/empty-worksapce-smoke-test-on-minikube-ubi8.yaml

```diff
@@ -32,57 +34,39 @@ jobs:
     - name: Checkout
       uses: actions/checkout@master
 
-    - name: Login to Quay
-      uses: docker/login-action@v2
-      with:
-        registry: quay.io
-        username: ${{ secrets.QUAY_USERNAME }}
-        password: ${{ secrets.QUAY_PASSWORD }}
-
+      # obtain the PR number for tegging the image
     - name: Get PR number
       id: get_pr_number
       run: |
         pr_number=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')
         echo "PR_NUMBER=$pr_number" >> $GITHUB_ENV
+        echo ">>>>>>>>>>>$pr_number"
+
+    - name: Start minikube cluster
+      id: run-minikube
+      uses: che-incubator/setup-minikube-action@next
+      with:
+        minikube-version: v1.31.0
 
+     # connect with docker daemon in the minikube and build an image there
+     # we need to build the image in the minikube because we have just 14 GB of space on the runner
+     # the UBI have more than 9 GB size this approach saves the disk space
     - name: Build base image
       run: |
-        cd base/ubi8 && docker buildx build --platform linux/amd64 -t quay.io/devfile/base-developer-image:${{ env.PR_NUMBER }} .
+       eval $(minikube docker-env)
+       cd base/ubi8 && docker buildx build --platform linux/amd64 -t quay.io/devfile/base-developer-image:${{ env.PR_NUMBER }} .
 
     - name: Build universal image
       run: |
-        cd universal/ubi8 && docker buildx build --platform linux/amd64 -t quay.io/devfile/universal-developer-image:${{ env.PR_NUMBER }} .
-
-    - name: Push universal base image
-      run: |
-        docker push quay.io/devfile/universal-developer-image:${{ env.PR_NUMBER }}
-
-    - name: Start minikube cluster
-      id: run-minikube
-      uses: che-incubator/setup-minikube-action@next
-      with:
-        minikube-version: v1.30.1
+        eval $(minikube docker-env)
+        cd universal/ubi8 && docker buildx build --platform linux/amd64 -t quay.io/devfile/base-developer-image:${{ env.PR_NUMBER }} .
```

Comment:
You are tagging this base image with the PR number. But the build of the universal image will use `ubi8-latest` as the base image, not the one that you just built ([FROM quay.io/devfile/base-developer-image:ubi8-latest](https://github.com/devfile/developer-images/blob/main/universal/ubi8/Dockerfile#L4C43-L4C54)). You should tag the base image with `ubi8-latest` too.

Reply:
Is it make sense to remove this job in this case? I guess this image can be pulled from the registry and we do not need to build locally.

Reply:
No. The PR can be about a change to the base image so we need to build both no?

Reply:
Ok, I just changed it on ubi8-latest. The build base image is here.